binaries/cli/src/main.rs

```diff
@@ -112,7 +119,12 @@ enum Command {
     /// Run runtime
     Runtime,
     /// Run coordinator
-    Coordinator { port: Option<u16> },
+    Coordinator {
+        #[clap(long, default_value_t = SocketAddr::new(
+            IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), DORA_COORDINATOR_PORT_DEFAULT)
+        )]
+        bind: SocketAddr,
```

Comment:
I feel like `bind` is not super clear, could we use `address` as the keyword for the address?

Reply:
I've used `bind` in the past to match the [TcpListener::bind](https://doc.rust-lang.org/std/net/struct.TcpListener.html#method.bind) call used to start the listener, but I'm fine with `addr` or `daemon_addr`.